jsk_arc2017_common/launch/object_estimation_by_scale.launch

```diff
@@ -0,0 +1,22 @@
+<launch>
+  <node name="scale0" pkg="jsk_arc2017_common" type="ekew_i_driver.py">
+    <rosparam>port: /dev/ttyUSB0</rosparam>
+  </node>
+  <node name="scale1" pkg="jsk_arc2017_common" type="ekew_i_driver.py">
+    <rosparam>port: /dev/ttyUSB1</rosparam>
+  </node>
+  <node name="scale2" pkg="jsk_arc2017_common" type="ekew_i_driver.py">
+    <rosparam>port: /dev/ttyUSB2</rosparam>
+  </node>
+  <node name="scale3" pkg="jsk_arc2017_common" type="ekew_i_driver.py">
+    <rosparam>port: /dev/ttyUSB3</rosparam>
```

Comment:
I think those scales should be pluged into new desktop PC.
If those scales are pluged into sheeta, we have to make udev rules to make symlink like Arduino